lib/ddtrace/contrib/active_record/events/sql.rb

```diff
@@ -39,6 +40,9 @@ def process(span, event, _id, payload)
             span.resource = payload.fetch(:sql)
             span.span_type = Datadog::Ext::SQL::TYPE
 
+            # Tag as an external peer service
+            span.set_tag(Datadog::Ext::Integration::TAG_PEER_SERVICE, span.service)
```

Comment:
I've deliberated if the peer service tag deserves its own first-class module, like analytics sampling rate has, but couldn't convince myself that having that module would add enough value to compensate for the creation of a trivial new file for us to maintain.
It would have simpler logic than analytics sampling rate: it would boil down to a getter and a setter on the already created constant `Datadog::Ext::Integration::TAG_PEER_SERVICE`.

I've leaned towards considering the constant, `Datadog::Ext::Integration::TAG_PEER_SERVICE`, as the main feature anchor for peer services and use it where needed, instead of creating a module.

But I'm still very much open to suggestions.